packages/client/src/routes.ts

```diff
@@ -23,18 +23,22 @@ export const geturl = (route: string, params: { slug: string }) => {
 
 type RoutePropExtended = RouteProps & { name: string };
 
+// XXX this should come from config
+// const slugString = ":slug(dakkapel-plaatsen|asdf)";
+const slugString = ":slug(dakkapel-plaatsen|asdf)";
```

Comment:
`|asdf` ?

Reply:
Yeah, so slug won't be any value (because of 404 discussion). slug will be a list of known slugs in `:slug(a|b|c)` format. Still needs to be implemented using something like this `topics.map(({slug}) => slug).join('|')`

Reply:
Let's discuss offline wednesday